src/components/AppRouterOutlet/AppRouterOutlet.tsx

```diff
@@ -18,6 +19,10 @@ export function AppRouterOutlet(): ReactElement {
         path="/processes/:processId"
         component={ProcessDetailPage}
       />
+      <PrivateRoute
+        path="/post/:personId/:definitionType"
```

Comment:
```suggestion
        path="/post/:personId/:definitionType/new"
```

this will allow for us to introduce a route to look at posts in the future if needed.